deployment/terraform/aws/main.tf

```diff
@@ -142,7 +142,7 @@ module "observers_2" {
 }
```

Comment:
Did we remove/modify https://github.com/zigbee-alliance/distributed-compliance-ledger/blob/master/deployment/README.md?